gpg.hcl

```diff
@@ -0,0 +1,19 @@
+binaries = ["bin/gpg", "bin/gpg-zip", "bin/gpgsplit", "bin/gpgv"]
+
+platform "darwin" "amd64" {
+  source = "https://github.com/cashapp/hermit-build/releases/download/gpg/gnupg-${version}-${os}-${arch}.tar.bz2"
+}
+
+platform "darwin" "arm64" {
```

Comment:
FYI you can just do `platform darwin {` here and drop the block above, but what you have will work fine.